ui/js/controllers/main.js

```diff
@@ -348,6 +356,10 @@ treeherderApp.controller('MainCtrl', [
             $location.search({"repo": repo_name});
         };
 
+        $scope.clearFilterBox = function() {
+            thJobFilters.removeSearchStrFilter();
+            document.getElementById("platform-job-text-search-field").focus();
```

Comment:
I wonder if we might use the same `$('#platform-job-text-search-field').focus();` that we do on L205, just for consistency.